tests/BaseTestCase.php

```diff
@@ -12,6 +11,20 @@
 
 class BaseTestCase extends TestCase
 {
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->withFactories(__DIR__ . '/database/factories');
+
+        $this->loadMigrationsFrom([
```

Comment:
why do we need a dummy table? Just curious 😄

Reply:
Because I switched it back from `make` to `create` for the factory creation the table needs to exist for the test